package.json

```diff
@@ -665,6 +665,15 @@
           "description": "Path to run neovim executable. For example, /usr/bin/nvim, or  C:\\Program Files\\Neovim\\bin\\nvim.exe",
           "default": "nvim"
         },
+        "vim.enableVimrc": {
+          "type": "boolean",
+          "description": "Use key mappings from a .vimrc file",
+          "default": "true"
+        },
+        "vim.vimrcPath": {
+          "type": "string",
+          "description": "Path to a Vim configuration file. If unset, it will check for $HOME/.vimrc or $HOME/_vimrc"
```

Comment:
What about `$XDG_CONFIG_HOME/nvim/init.vim`? (https://neovim.io/doc/user/starting.html#vimrc)

Feel free to disagree, but my I think my preferred approach for this is have `vim.vimrcPath` default to an empty string. Then note here that if it's unset, we fall back on default locations (like `$HOME/.vimrc`, `$HOME/_vimrc`, and `$XDG_CONFIG_HOME/nvim/init.vim`) and then link to the part of the `README` which lists these locations.

A nice-to-have would be a pop up the first time it defaults to a location (which is probably immediately after you enable this feature) that says something like "We couldn't find your .vimrc. Click **here** to set the location" or "We found a .vimrc here: /path/to/.vimrc. Click **here** to use it"

Reply:
> What about `$XDG_CONFIG_HOME/nvim/init.vim`? (https://neovim.io/doc/user/starting.html#vimrc)

I skipped this initially because `$XDG_CONFIG_HOME` wasn't set by default on my Ubuntu machine. But, I can certainly add it, plus the default paths from your link.

> Feel free to disagree, but my I think my preferred approach for this is have `vim.vimrcPath` default to an empty string. Then note here that if it's unset, we fall back on default locations (like `$HOME/.vimrc`, `$HOME/_vimrc`, and `$XDG_CONFIG_HOME/nvim/init.vim`) and then link to the part of the `README` which lists these locations.

I'm not sure I understand how this is different from what I did?

> A nice-to-have would be a pop up the first time it defaults to a location (which is probably immediately after you enable this feature) that says something like "We couldn't find your .vimrc. Click **here** to set the location" or "We found a .vimrc here: /path/to/.vimrc. Click **here** to use it"

I like this idea! I'll see what I can do.

Reply:
> I'm not sure I understand how this is different from what I did?

Sorry about that - I think we should support more default locations and list them externally (with a link in the description).